packages/esm-patient-forms-app/src/forms/form-view.component.tsx

```diff
@@ -82,54 +82,81 @@ const FormView: React.FC<FormViewProps> = ({ forms, patientUuid, patient, pageSi
     [results],
   );
 
+  const handleFilter = ({ rowIds, headers, cellsById, inputValue, getCellId }: FilterProps): Array<string> => {
+    return rowIds.filter((rowId) =>
+      headers.some(({ key }) => {
+        const cellId = getCellId(rowId, key);
+        const filterableValue = cellsById[cellId].value;
+        const filterTerm = inputValue.toLowerCase();
+
+        return ('' + filterableValue).toLowerCase().includes(filterTerm);
+      }),
+    );
+  };
+
+  if (!forms?.length) {
+    return (
+      <Layer>
```

Comment:
No, because we're not using that EmptyState here, rather we're using a custom empty state.

Reply:
I believe we can take this change to the common lib's EmptyState and put another prop (say `dataType` (here 'category')) and give this prop a default value of "data", so that this change is can be re-usable.
Thanks!

Reply:
The reason that the EmptyState component won't work as well here is that it renders a header title wrapped in an `h4` tag inside the tile, which works well for widget cards. It wouldn't work for this use case unless we modify it to remove the `h4`. Modifying the EmptyState component to get it to work for this one use case is unnecessary, which is likely why the original author elected to use their own custom empty state.

Reply:
Got it.
I forgot about the header. Thanks for the help!